src/test/java/apitests/retrofit/TrelloBoardTest.java

```diff
@@ -0,0 +1,168 @@
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.testng.Assert.assertEquals;
+
+public class TrelloBoardTest {
+
+    String key;
+    String token;
+    Retrofit retrofit;
+    List<String> idListOfCreatedBoards;
+
+    @BeforeClass
+    public void init(){
+        idListOfCreatedBoards = new ArrayList<>();
+        Properties properties = new Properties();
+        String pathToFile = "src/main/resources/trello.properties";
+
+        try {
+            properties.load(new FileInputStream(pathToFile));
+            key = properties.getProperty("key");
+            token = properties.getProperty("token");
+        }
+        catch (IOException ex){
+            ex.printStackTrace();
+        }
+
+        OkHttpClient.Builder httpClient = new OkHttpClient.Builder();
+        retrofit = new Retrofit.Builder()
+                .baseUrl("https://trello.com")
+                .addConverterFactory(GsonConverterFactory.create())
+                .client(httpClient.build())
+                .build();
+    }
+
+    @AfterClass
+    public void clean() throws IOException {
+        BoardService service = retrofit.create(BoardService.class);
+        Call<BoardForTests> callSync;
+        for (String id : idListOfCreatedBoards){
+            callSync = service.deleteBoard(id, key, token);
+            callSync.execute();
+        }
+    }
+
+    @Test
+    public void creatingNewBoardTest() throws IOException {
+        BoardForTests board = new BoardForTests();
+        board.setName("NewBoard");
+        board.setDesc("This is new board!");
+        board.setClosed(false);
+
+        BoardService service = retrofit.create(BoardService.class);
+        Call<BoardForTests> callSync = service.postBoard(key, token, board);
+
+        Response<BoardForTests> response = callSync.execute();
+        BoardForTests createdBoard = response.body();
+
+        if (createdBoard != null) {
+            idListOfCreatedBoards.add(createdBoard.getId());
+        }
+
+        assertEquals(response.code(), 200);
+        assertThat(createdBoard).hasFieldOrPropertyWithValue("name", "NewBoard")
+                    .hasFieldOrPropertyWithValue("desc", "This is new board!")
+                    .hasFieldOrPropertyWithValue("closed", false);
+
+    }
+
+    @Test
+    public void gettingBoardByIDTest() throws IOException {
+        BoardForTests board = new BoardForTests();
+        board.setName("NewBoard");
+        board.setDesc("This is new board!");
+        board.setClosed(false);
+
+        BoardService service = retrofit.create(BoardService.class);
+        Call<BoardForTests> callSyncPost = service.postBoard(key, token, board);
+
+        Response<BoardForTests> responsePost = callSyncPost.execute();
+        BoardForTests createdBoard = responsePost.body();
+
+        if (createdBoard != null) {
```

Comment:
with this code, if createdBoard will be null, no assertions will be run and the test will pass.
as above - assert that created board is not null, add it to the list and make rest of the assertions below